scripts/MRK17_Exp1_rk01.jmd

```diff
@@ -122,11 +123,11 @@ m2form_b = @formula (-1000/rt) ~ 1 + f*p*q*lq*lt +
  (1 | Subj) + (0+f | Subj) + (0+p | Subj) + (0+q | Subj) + (0+lq | Subj) + (0+lt | Subj) +
  (1 | Item) +                (0+p | Item) + (0+q | Item) + (0+lq | Item) + (0+lt | Item);
 
-zcpLMM_b = @time fit(LinearMixedModel, m2form_b, dat, contrasts=contrasts);
+zcpLMM_b = fit(LinearMixedModel, m2form_b, dat, contrasts=contrasts);
 
-const mods = [cmplxLMM, zcpLMM, zcpLMM_b];
+mods = [cmplxLMM, zcpLMM, zcpLMM_b];
 gof_summary = DataFrame(dof=dof.(mods), deviance=deviance.(mods),
-              AIC = aic.(mods), AICc = aicc.(mods), BIC = bic.(mods))
+              AIC = aic.(mods), AICc = aicc.(mods), BIC = bic.(mods)) # this has no effect
```

Comment:
It has no effect in the Jupyter notebook because it is not the last expression in the code cell.  Either split this cell into two cells or drop the declaration.